src/styles/episodic/site-base.css

```
@@ -0,0 +1,285 @@
+  margin: 0;
+  /* A fine grid, at the threshold of visibility: the ruled paper the
+     original specification sheet was drawn on. */
+  background:
+    linear-gradient(rgba(255, 255, 255, 0.014) 1px, transparent 1px),
+    linear-gradient(90deg, rgba(255, 255, 255, 0.014) 1px, transparent 1px), var(--color-base-100);
+  background-size: 3rem 3rem;
+  color: var(--color-base-content);
+  font-family: var(--font-sans);
+  font-size: 1.05rem;
+  line-height: 1.6;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.14.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)